index.html

```diff
@@ -5,9 +5,50 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="css/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Acme&family=Fredoka:wght@300;400;500&family=Mukta:wght@300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Plantcare</title>
   </head>
   <body>
-    <p class="test">OLa</p>
+    <header class="header">
+      <p class="logo">Plant<span>Care</span></p>
+      <nav class="header__navigation">
+        <ul class="header__navigation--list">
+          <li class="header__navigation--item">
+            <a href="#" class="header__navigation--link">Text 1</a>
```

Comment:
Why  are you using this `Text x` as links. Dint we know what will be the proper name yet?

Reply:
Exactly, at the time of this commit, we were still deciding what was the text to put in these elements. But there is already a new pull request with the text updated.